src/tiled/tilesetdock.cpp

```diff
@@ -553,6 +576,11 @@ void TilesetDock::deleteTilesetView(int index)
     delete view;                    // view needs to go before the tab
     mTabBar->removeTab(index);
 
+    // Make the "New Tileset..." special tab reappear if there is no tileset open
+    if (mTilesets.count() == 0) {
```

Comment:
Can remove the braces since it's only 1 line block.

Reply:
I've already changed that in the second commit.